### DISCUSSION TITLE
Override default color

### DISCUSSION THREAD

**Discussion by stedekay** [tailwindlabs/tailwindcss#2029] [Help] [answered]
Hi, 
we want to build some standard components like the following:

```css
.tile {
    @apply p-1 bg-blue-100 my-px;
}

.tile:hover {
    @apply bg-blue-200;
}
```

Sometimes we want to override the color, so for example:
```html
<div class="tile bg-red-100 hover:bg-red-200">
 ...
</div>
```

But in this case the hover is still blue.

As we want to be flexible with the colors we don't want to add classes like .tile-red, .tile-green etc. like in the example of the buttons but we stillt want to leave the default as blue. Are there some alternatives how to accomplish this?

**Comment by Apostolique** [comment]
I believe you can put your tile definition above the utilities. That way the utilities will take precedence.

```css
@tailwind base;

@tailwind components;

.tile {
    @apply p-1 bg-blue-100 my-px;
}

.tile:hover {
    @apply bg-blue-200;
}

@tailwind utilities;
```

**Reply by stedekay** [reply]
This is already the case.

**Reply by stedekay** [reply]
Okay, seems to be a config issue as my color red-200 isn't available. So thanks for your input, definitely learned something from your answer.

**Comment by stedekay** [comment, marked as answer]
This was a misconfiguration issue.

**Reply by Apostolique** [reply]
What was the issue? It was getting purged?

**Reply by stedekay** [reply]
We were overriding the color palettes and it looked like this:
```js
red: {
    100: '#FCBCB4',
    300: '#ff0000',
},
```

:facepalm:
